src/style.css

```diff
@@ -0,0 +1,123 @@
+
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+  font-weight: 300;
+  padding-top: 20px ;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+input {
+  padding: 0;
+}
+
+.form-big {
+  margin-top: 0;
+  margin-bottom: 20px;
+}
+
+.form-small {
+  margin-top: 0;
+}
+
+.search {
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
apply `box-sizing` to the all elements or remove it. It is confusing when there is a difference in size calculating approach